step.sh

```diff
@@ -412,6 +421,13 @@ else
 			-c "${compile_bitcode}" \
 			-t "${team_id}"
 		cd "${curr_pwd}"
+	else
+		echo_info "Using custom export options plist..."
+		echo
+		echo "$custom_export_options_plist_content"
+		echo
+
+		echo "$custom_export_options_plist_content" >> "$export_options_path"
```

Comment:
echo "$custom_export_options_plist_content" >> "$export_options_path" -> `echo "$custom_export_options_plist_content" > "$export_options_path"`

do not append!